src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -30,7 +30,7 @@ New Relic has several [APIs](/docs/apis/get-started/intro-apis/introduction-new-
 
 NerdGraph is built using [GraphQL](https://graphql.org), which is an open source API format that allows you to request exactly the data needed, with no over-fetching or under-fetching.
 
-For a lesson in how to use NerdGraph, watch this 7-minute video:
+Here's a short video with an introduction to using NerdGraph and the explorer. 
```

Comment:
@zuluecho9 we don't seem to explain explorer at all in the intro; yet, we mention it and start right in with the video. We define explorer in `src/content/docs/apis/nerdgraph/get-started/nerdgraph-explorer.mdx`. Consider adding even a short explanation of explorer in the introduction since you mention it.